Sources/Workspace/DefaultPluginScriptRunner.swift

```diff
@@ -99,9 +96,15 @@ public struct DefaultPluginScriptRunner: PluginScriptRunner {
         command += self.toolchain.swiftCompilerFlags
 
         command += ["-swift-version", toolsVersion.swiftLanguageVersion.rawValue]
-        command += ["-I", runtimePath.pathString]
+        // if runtimePath is set to "PackageFrameworks" that means we could be developing SwiftPM in Xcode
+        // which produces a framework for dynamic package products.
+        if runtimePath.basename == "PackageFrameworks" {
+            command += ["-I", runtimePath.parentDirectory.pathString]
```

Comment:
@abertelrud not quite sure why we have to walk one directory up when using the frameworks in "Xcode mode", maybe there is a better way?